Framework/Core/Xml/System/Xml/XmlWriter.cs

```diff
@@ -0,0 +1,1030 @@
+            {
+                try
+                {
+                    if (_Position > 0)
+                        _Stream.Write(_Buffer, 0, _Position);
+                }
+                catch (Exception e)
+                {
+                    throw new IOException("Internal FlushBuffer() _Stream.Write", e);
+                }
+
+                _Position = 0;
+            }
+
+            return;
+        }
+
+        //
+        // Summary:
+        //     When overridden in a derived class, returns the closest prefix defined in
+        //     the current namespace scope for the namespace URI.
+        //
+        // Parameters:
+        //   ns:
+        //     The namespace URI whose prefix you want to find.
+        //
+        // Returns:
+        //     The matching prefix or null if no matching namespace URI is found in the
+        //     current scope.
+        //
+        // Exceptions:
+        //   System.ArgumentException:
+        //     ns is either null or String.Empty.
+        public string LookupPrefix(string ns)
+        {
+            if (ns == null)
+                throw new ArgumentException("must not be null", "ns");
+            return "";
+            // need to add namspace list lookup
+        }
+
+        //
+        // Summary:
+        //     When overridden in a derived class, writes out the attribute with the specified
+        //     local name and value.
+        //
+        // Parameters:
+        //   localName:
+        //     The local name of the attribute.
+        //
+        //   value:
+        //     The value of the attribute.
+        //
+        // Exceptions:
+        //   System.InvalidOperationException:
+        //     The state of writer is not WriteState.Element or writer is closed.
+        //
+        //   System.ArgumentException:
+        //     The xml:space attribute value is invalid.
+        public void WriteAttributeString(string localName, string value)
+        {
+            this.WriteAttributeString(null, localName, null, value);
+        }
+
+        //
+        // Summary:
+        //     When overridden in a derived class, writes an attribute with the specified
+        //     local name, namespace URI, and value.
+        //
+        // Parameters:
+        //   localName:
+        //     The local name of the attribute.
+        //
+        //   value:
+        //     The value of the attribute.
+        //
+        //   ns:
+        //     The namespace URI to associate with the attribute.
+        //
+        // Exceptions:
+        //   System.InvalidOperationException:
+        //     The state of writer is not WriteState.Element or writer is closed.
+        //
+        //   System.ArgumentException:
+        //     The xml:space attribute value is invalid.
+        public void WriteAttributeString(string localName, string ns, string value)
+        {
+            this.WriteAttributeString(null, localName, ns, value);
+        }
+
+        //

```

Comment:
If it's low-hanging fruit, yes.  It may also be worth auditing the comments for accuracy (as you've done with the examples above), but this is probably quite a bit more expensive to do on a broad scale.